src/config/authd-config.c

```diff
@@ -187,3 +281,32 @@ short eval_bool(const char *str) {
         return OS_INVALID;
     }
 }
+
+int Test_Authd(const char *path, char **output) {
+    authd_config_t *test_authd;
+    os_calloc(1, sizeof(authd_config_t), test_authd);
+
+    if (ReadConfig(CAUTHD, path, test_authd, NULL, output) < 0) {
+        if (output == NULL){
+            merror(CONF_READ_ERROR, "Authd");
+        } else {
+            wm_strcat(output, "ERROR: Invalid configuration in Authd", '\n');
+        }
+        free_authd_config(test_authd);
```

Comment:
This sentence is the same as line 300. Please, modify it so that it doesn't repeat itself.